README.md

```diff
@@ -1,18 +1,30 @@
-## cached-io: cache a single IO action
+# cached-io: cache a single IO action
 
 Sometimes you have an action that does something really expensive
 whose results don't change that much. This is a simple library that
 lets you cache the output of that expensive action for a
 developer-specified length of time.
 
-This library is slightly more powerful than
-[io-memoize](https://hackage.haskell.org/package/io-memoize) which
-lets you memoize the action forever.
+`test/test-cachedIO.hs` shows a very basic usage example.
 
-### Example
+## Developing this library
 
-See test/test-cachedIO.hs.
+### Formatters
 
-#### Test
+The formatters used in this repo are provided by `shell.nix`:
 
-Testing Jenkins stuff.
+- `*.hs`: [`ormolu`](https://github.com/tweag/ormolu)
+- `*.cabal`:
+  [`cabal-fmt`](https://hackage.haskell.org/package/cabal-fmt)
+  (`cabal-fmt --inplace aws-arn.cabal`)
+- `*.nix`:
+  [`nixpkgs-fmt`](https://github.com/nix-community/nixpkgs-fmt)
+  (`nixpkgs-fmt *.nix`)
+
+### Regenerate CI
+
+This repo uses `haskell-ci`, which is provided by `shell.nix`:
```

Comment:
This repo doesn't have a `flake-compat` setup, so either recommend `nix develop` or add flake-compat support?

If you want to add flake-compat: steal `default.nix` and `shell.nix` from aws-arn and add the flake-compact input (also steal from aws-arn, though it's always copy-paste).